diff --git a/test/test-list.h b/test/test-list.h
index b79a4ea..3e13150 100644
--- a/test/test-list.h
+++ b/test/test-list.h
@@ -464,7 +464,6 @@ TEST_FS_DECLARE   (fs_wtf)
 #endif
 TEST_FS_DECLARE   (fs_get_system_error)
 TEST_DECLARE   (strscpy)
-TEST_DECLARE   (strtok)
 TEST_DECLARE   (threadpool_queue_work_simple)
 TEST_DECLARE   (threadpool_queue_work_einval)
 TEST_DECLARE   (threadpool_multiple_event_loops)
@@ -1192,7 +1191,6 @@ TASK_LIST_START
   TEST_ENTRY  (get_osfhandle_valid_handle)
   TEST_ENTRY  (open_osfhandle_valid_handle)
   TEST_ENTRY  (strscpy)
-  TEST_ENTRY  (strtok)
   TEST_ENTRY  (threadpool_queue_work_simple)
   TEST_ENTRY  (threadpool_queue_work_einval)
   TEST_ENTRY_CUSTOM (threadpool_multiple_event_loops, 0, 0, 60000)
